corehq/motech/views.py

```diff
@@ -116,6 +90,120 @@ def page_url(self):
         return reverse(self.urlname, args=[self.domain, pk])
 
 
+@login_or_api_key
+@require_permission(Permissions.edit_motech)
+def motech_log_export_view(request, domain):
+    """
+    Download ``RequestLog``s as CSV. Uses ``StreamingHttpResponse`` to
+    support large file sizes.
+
+    CSV files can be imported into PostgreSQL using ::
```

Comment:
I don't think this is useful info in this docstring, but could instead go in user facing docs if you think necessary.

Reply:
The partner motivating for this specifically asked how to automate getting data into Superset. So I think once this is available, it might be useful to move this to somewhere accessible to partners who would benefit from it ... but maybe harder to find for people who wouldn't use it? Maybe under the [Confluence page on data forwarding](https://confluence.dimagi.com/pages/viewpage.action?pageId=12224128) or under [Integrations on ReadTheDocs](https://commcare-hq.readthedocs.io/api.html)?